.changeset/red-candles-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/core': patch
+---
+
+Add a guard to "maskTypenames" so a null value isn't considered an object.
```

Comment:
Just for other changesets, let’s write them more in the mood of why sth has changed rather than what has changed :)